src/sql/src/kafka_util.rs

```diff
@@ -516,8 +518,14 @@ pub fn generate_ccsr_client_config(
     }
 
     let mut ccsr_options = extract(
-        &mut ccsr_options,
-        &[Config::string("username"), Config::string("password")],
+        ccsr_options,
+        &[
+            // An old migration
+            // (https://github.com/MaterializeInc/materialize/blob/c402917b7078a14bc0d0d6330b9c9b177aa73e47/src/coord/src/catalog/migrate.rs#L362)
+            // adds this field to kafka-avro WITH options, though now in the CSR case,
+            // the field is unread and fixed to `true`
```

Comment:
That was just an error in the migration, no? Like this parameter has no meaning here and has never had meaning, right? Too bad. If that's true we should write another migration to fix it. Doesn't need to block this though since this is still a huge net win. Can you file a followup ticket though?

Reply:
I am not sure if it never had meaning, but yeah, now It appears to not have meaning...I will make an issue

also, I removed the `username` and `password` extraction, because I don't actually think those exist except in docs?